docs-app/ember-cli-build.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const { Webpack } = require('@embroider/webpack');
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function (defaults) {
+  const app = new EmberApp(defaults, {
+    // Add options here
+    autoImport: {
+      watchDependencies: ['ember-container-query'],
+    },
+    'ember-cli-babel': {
+      enableTypeScriptTransform: true,
+    },
+  });
+
+  return require('@embroider/compat').compatBuild(app, Webpack, {
+    packagerOptions: {
+      webpackConfig: {
+        // ...
+      },
+    },
+    skipBabel: [
+      {
+        package: 'qunit',
+      },
+    ],
+    splitAtRoutes: ['album', 'dashboard', 'form', 'products'],
+    staticAddonTestSupportTrees: true,
+    staticAddonTrees: false, // due to ember-css-modules
+    staticComponents: false, // due to ember-css-modules
```

Comment:
**TODO**

If no further development to make `ember-css-modules` compatible with Embroider happens, consider removing the addon.

https://github.com/evoactivity/ember-modern-css#ember-modern-css